index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="kr">
```

Comment:
❗️ 수정요청

html lang 속성은 스크린리더, 검색 엔진등에게 해당 사이트가 어떤 언어 기반으로 작성되었는지 설명하는 속성입니다.
해당 속성에는 이미 특정 언어들을 의미하는 예약어가 들어가야 합니다. 예를들어 en은 english 즉 영어를 의미합니다.
kr은 신기하게도 Kanuri 어를 의미합니다. 한국어는 ko를 써야합니다~

https://developer.mozilla.org/ko/docs/Web/HTML/Global_attributes/lang
(해당 페이지 sidebar LockUp 에서 kr과 ko를 검색해보세요~)
https://r12a.github.io/app-subtags/

```suggestion
<html lang="ko">
```